src/components/auth/SignInScreen.js

```diff
@@ -103,104 +103,138 @@ export default class SignInScreen extends React.Component {
                   {/* Sign In Button */}
                   <TouchableOpacity
                     onPress={() => this.signIn()}
                     disabled={this.state.isLoading}
                     style={styles.buttonStyle1}>
+                  {!this.state.isLoading &&
                     <Text style={styles.buttonText1}>
-                      {i18n.t('Sign In')}
-                    </Text>
-                  </TouchableOpacity>
-                  {/* Sign Up Text */}
-                  <TouchableOpacity
-                    onPress={() => this.props.navigation.navigate('SignUp')}
-                    style={styles.buttonStyle2}>
-                    <Text style={styles.buttonText2}>
-                      {i18n.t('New User? Create an Account!')}
-                    </Text>
-                  </TouchableOpacity>
-                  {/* Forgot Password Text */}
-                  <TouchableOpacity
-                    onPress={() => this.props.navigation.navigate('ForgotPassword')}
-                    style={styles.buttonStyle2}>
-                    <Text style={styles.buttonText2}>
-                      {i18n.t('Forgot Password?')}
+                      Entrar
                     </Text>
-                  </TouchableOpacity>
-                  {/* Loading ActivityIndicator */}
+                  }
                   {this.state.isLoading &&
                     <View>
-                      <ActivityIndicator color={COLORS.lightblue} size='large' animating={this.state.isLoading} />
+                      <ActivityIndicator color={COLORS.sandy} size='large' animating={this.state.isLoading} />
                     </View>
                   }
-                </View>
+                  </TouchableOpacity>
+                </KeyboardAvoidingView>
               </Container>
-            </View>
           </TouchableWithoutFeedback>
-        </KeyboardAvoidingView>
+        <MenuBar navigation = {this.props.navigation}/>
       </SafeAreaView>
     )
   }
 }
+
+const { width, height } = Dimensions.get('screen');
+
 const styles = StyleSheet.create({
+
+  backgroundContainer: {
+    width: width,
+    height: height,
+    backgroundColor: COLORS.sandy,
+    justifyContent: 'flex-end'
+  },
+
   container: {
     flex: 1,
-    backgroundColor: COLORS.lightgreen,
+    backgroundColor: COLORS.sandy,
     justifyContent: 'center',
-    flexDirection: 'column'
+    flexDirection: 'column',
   },
+
+  container1: {
+    flex: 2,
+    backgroundColor: 'transparent',
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+    flexDirection: 'column',
+    paddingHorizontal: 5,
+  },
+
+  container2: {
+    flex: 1,
+    backgroundColor: 'transparent',
+    justifyContent: 'flex-end',
+    flexDirection: 'column',
+  },
+
   input: {
     flex: 1,
-    fontSize: 17,
-    fontWeight: 'bold',
-    color: COLORS.lightblue,
+    fontSize: 22,
+    fontWeight: 'normal',
+    color: COLORS.forestgreen,
   },
+
   infoContainer: {
```

Comment:
## refer to [Scaling](https://github.com/wang1kevin1/InstitutoTerra/pull/43/files#r451776397)